### ISSUE TITLE
Applying to add a paper

### ISSUE THREAD

**Issue by tian-qing001** [cmhungsteve/Awesome-Transformer-Attention#63]
Hi~ Thanks for maintaining this awesome repository!

We have a new paper [Agent Attention](https://arxiv.org/abs/2312.08874) proposing a novel attention paradigm, which we think is relevant to the [General Vision Transformer](https://github.com/cmhungsteve/Awesome-Transformer-Attention?tab=readme-ov-file#general-vision-transformer) part of your repository. We are wondering if you could please add our paper to the collection.

Agent Attention: "Agent Attention: On the Integration of Softmax and Linear Attention", arXiv, 2023 (Tsinghua). [[Paper](https://arxiv.org/abs/2312.08874)][[PyTorch](https://github.com/LeapLabTHU/Agent-Attention)]

Thank you very much.

**Comment by cmhungsteve** [comment]
Added.
Thank you for sharing
